bindings/nodejs/src/connection.rs

```diff
@@ -241,21 +116,11 @@ impl Connection {
     /// ```
     #[napi(catch_unwind)]
     pub async fn execute(&self, query: String) -> napi::Result<()> {
-        let sess = self.session.clone();
-        let mut sess = sess.lock().await;
-
-        let plan = sess
-            .create_logical_plan(&query)
+        self.inner
+            .execute(query)
```

Comment:
`.execute.execute` seems like it shouldn't be valid.

Reply:
yeah, known issue, this is a rust sdk problem that I talked about with sean earlier. The API isn't public (yet) an I'm trying to figure out the right words for these things.